R/constants.R

```diff
@@ -19,6 +19,7 @@ FLAG_PROTECT_FLOAT <- "FLAG_PROTECT_FLOAT"
 # Time constants
 HISTORICAL_YEARS <- 1971:2010
 FUTURE_YEARS <- seq( 2015, 2100, 5 )
+IMF_FUTURE_YEARS <- seq( 2010, 2020, 1 )
```

Comment:
I just happened to notice this from the code snippet above--I don't like the "IMF_FUTURE_YEARS" object 2010:2020, because within that dataset, the "future" years differ by nation, and for most the 2010:2014 time period is actual historical data, with 2015:2020 being the future years. What do you guys think about just calling it IMF_YEARS?